src/reducers/linkages.js

```diff
@@ -5,26 +5,25 @@ import {
   REMOVE_LINKAGE,
 } from 'actions';
 
+import { dissoc, path, assoc } from 'ramda';
+
+const getLinkagesFromAction = action =>
```

Comment:
In `ramda` all functions are curried so, you can simplify this method like this:

```js
const getLinkagesFromAction = 
  path(['payload', 'state', 'network', 'linkages']);
```